server/charts/historian/values.yaml

```diff
@@ -35,6 +35,8 @@ historian:
   system:
     httpServer:
       connectionTimeoutMs: 0
+  config:
```

Comment:
I think this would not work with the current changes in the configmap. In the configmap, we have:

```
"config": {
            "configDumpEnabled": {{ .Values.config.configDumpEnabled }}
        },
```

But here, the path for these values is:
Values.historian.config.configDumpEnabled
Values.gitrest.config.configDumpEnabled

To fix this, you either need to update the configmaps to add a `historian.` or `.gitrest.` prefix in the value path, or move his setting to the top level. I prefer the former approach, meaning you just need to edit the configmaps.